index.html

```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
         </ul>
     </ul>
   </section>
+  <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
```

Comment:
- [ ] Kindly layout the elements in this container with flexbox, it makes our work easier to design and build responsive web pages without having to use tricky hacks. ℹ️